modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/VersionsContent.tsx

```diff
@@ -48,8 +50,9 @@ const VersionsContent = ({
 				throw new Error(`Failed to fetch ${getItemVersionsURL}`);
 			}
 
-			const {versions}: IData = await response.json();
+			const {versions, viewVersionsURL}: IData = await response.json();
```

Comment:
Maybe we can use a versionsData object and use only one `useState` hook, what do you think?